### ISSUE TITLE
Cannot read property 'browse' of undefined

### ISSUE THREAD

**Issue by NetOperatorWibby** [silverwind/updates#12]
```
TypeError: Cannot read property 'browse' of undefined
    at getInfoUrl (~/node_modules/updates/updates.js:174:54)
    at Promise.all.then.dati (~/node_modules/updates/updates.js:192:30)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

I'm getting this in my projects when I run `updates --update ./` My `updates` version is `^5.3.0`.

**Comment by silverwind** [comment]
Should be fixed in 5.3.1.

**Comment by NetOperatorWibby** [comment]
Sweet, it works again! Thanks for this module, I love it.
